assets/js/blocks/mini-cart/style.scss

```diff
@@ -29,7 +31,7 @@
 @media screen and (min-width: 768px) {
 	.wc-block-mini-cart__amount {
 		display: initial;
-		font-weight: 600;
+		font-weight: inherit;
```

Comment:
If the `font-weight` is being set in the base ruleset above, this line doesn't seem to be needed.

Although, it may make sense to declare both the font weight and size here instead, since we're adjusting the display property. What do you think?

Reply:
@danielwrobert, thanks for catching this! With [b838f00](https://github.com/woocommerce/woocommerce-blocks/pull/7515/commits/b838f006960f97d176bb944250562596f81553bc) should be fixed 🚀